pkg/types/consensus_state.go

```diff
@@ -4,12 +4,12 @@ package types
 import (
 	"encoding/base64"
 	"encoding/hex"
+	"gopkg.in/yaml.v2"
```

Comment:
I got check failure from `golangci-lint`

Are you sure the previous import was not right?